src/version.rs

```diff
@@ -0,0 +1,23 @@
+use std::env;
+
+// include version into the binary at compile time
```

Comment:
It is very important to read the version env var at compile time so that it's hardcoded into the final binary for predictability and reading the library version and not the version of the application using this library